File: Sources/2908/2908.java
/**
 *  BOJ 2908번 Java 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 14,260 KB / 278,528 KB
 *  소요 시간 : 112 ms / 3,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

import java.util.*;
public class Main{
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		int A=rev(sc.nextInt());
		int B=rev(sc.nextInt());
		System.out.println(A>B?A:B);
		sc.close();
	}
	private static int rev(int i){
		return Integer.parseInt(new StringBuilder(Integer.toString(i)).reverse().toString());
	}
}

File: Sources/2908/2908.kt
/**
 *  BOJ 2908번 Kotlin (Native) 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 14,252 KB / 147,456 KB
 *  소요 시간 : 0 ms / 1,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

fun main(args:Array<String>){
	var nums=readLine()!!.split(" ")
	var A=nums[0].reversed().toInt()
	var B=nums[1].reversed().toInt()
	println(kotlin.math.max(A,B))
}